apps/ftu/js/tutorial.js

```diff
@@ -298,14 +236,15 @@
 
       var stepPromise;
       if (stepData.video) {
-        stepPromise = _loadMedia(videoElement, stepData.video).then(function(){
-          videoElement.play();
+        stepPromise = TutorialUtils.getBestAssetForDirection(stepData.video)
```

Comment:
This is the actual functional change - using getBestAssetForDirection to determine if we can get a direction-specific video (e.g. Homescreen-rtl.mp4 vs Homescreen.mp4) or just use what is in the config